src/tests/integration-test

```diff
@@ -1746,6 +1754,12 @@ if __name__ == '__main__':
                            help='name of test class or method (e. g. "Drive", "FS.test_ext2")')
     cli_args = argparser.parse_args()
 
+    distro, version = get_distro_version()
+    if distro in ('centos', 'enterprise_linux'):
+        print('Skipping integration tests on "%s %s".'\
+              'SCSI debug devices causing kernel panic.' % (distro, version))
+        sys.exit(1)
```

Comment:
I'm afraid this has to return ``0``, otherwise the tests will be always marked as failed.